helix-core/src/main/java/org/apache/helix/model/ClusterConfig.java

```diff
@@ -134,7 +134,7 @@
     // offline for more than this specified time period, and users call purge participant API,
     // then the node will be removed.
     // The unit is milliseconds.
-    OFFLINE_NODE_TIME_OUT_FOR_PURGE
+    OFFLINE_DURATION_FOR_PURGE
```

Comment:
Optional: usually I see people do `OFFLINE_DURATION_FOR_PURGE_MS` but either way is fine, really.